discover_overlay/locales/fr/LC_MESSAGES/default.po

```diff
@@ -109,13 +109,13 @@ msgid "Right"
 msgstr "Droite"
 
 msgid "Settings"
-msgstr "Paramètres"
+msgstr "Ouvrir les Paramètres"
```

Comment:
That would translate to "open settings" but the source string is just "settings" so I would leave it as is.

Reply:
Actually, this one is probably the more egregious issue in the English translations mentioned here. We've used 'Config', 'Configure', 'Configuration' and 'Settings' interchangeably, which could be confusing.

I'd be happy for this to be 'Open Configuration' or similar.